llvm/lib/Transforms/Vectorize/VPlanTransforms.cpp

```diff
@@ -1467,6 +1467,49 @@ void VPlanTransforms::truncateToMinimalBitwidths(
          "some entries in MinBWs haven't been processed");
 }
 
+/// Remove BranchOnCond recipes with true conditions together with removing
+/// dead edges to their successors.
+static void simplifyCFG(VPlan &Plan) {
```

Comment:
Perhaps a more modest name would be more accurate, at this stage?

Reply:
Updated to simplifyBranchOnCondTrue, thanks